module/apmfiber/middleware_test.go

```diff
@@ -0,0 +1,310 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package apmfiber_test
```

Comment:
```suggestion
//go:build go1.12
// +build go1.12

package apmfiber_test
```

Reply:
We need to exclude these tests on older versions of Go, as it causes CI to fail:

https://apm-ci.elastic.co/blue/organizations/jenkins/apm-agent-go%2Fapm-agent-go-mbp/detail/PR-999/30/pipeline

Reply:
It will also be necessary to add a `doc.go` file which does _not_ have this build tag, to make CI happy.

Reply:
added build tag in _test.go and doc.go